screens/Providers.js

```diff
@@ -54,5 +74,23 @@ const styles = StyleSheet.create({
     container: {
         flex: 1,
         backgroundColor: "#9bcdd5",
+    },
+    addButton: {
+        backgroundColor: 'white',
+        padding: 10,
+        width: 110,
+        marginBottom: 10,
+        borderRadius: 10,
+        left:10
+    },
+    backButton: {
```

Comment:
This style could also be added to GlobalStyle since we use it in at least 3 files. Would help us keep our code DRY.